dev/tooltip.html

```diff
@@ -32,24 +33,36 @@
       }
     </style>
 
+    <vaadin-radio-group id="positionGroup" label="Tooltip Position" theme="vertical" value="bottom">
+      <vaadin-radio-button value="bottom-start" label="Bottom Start"></vaadin-radio-button>
+      <vaadin-radio-button value="bottom" label="Bottom"></vaadin-radio-button>
+      <vaadin-radio-button value="bottom-end" label="Bottom End"></vaadin-radio-button>
+      <vaadin-radio-button value="start-top" label="Start Top"></vaadin-radio-button>
+      <vaadin-radio-button value="start" label="Start"></vaadin-radio-button>
+      <vaadin-radio-button value="start-bottom" label="Start Bottom"></vaadin-radio-button>
+      <vaadin-radio-button value="top-start" label="Top Start"></vaadin-radio-button>
+      <vaadin-radio-button value="top" label="Top"></vaadin-radio-button>
+      <vaadin-radio-button value="top-end" label="Top End"></vaadin-radio-button>
+      <vaadin-radio-button value="end-top" label="End Top"></vaadin-radio-button>
+      <vaadin-radio-button value="end" label="End"></vaadin-radio-button>
+      <vaadin-radio-button value="end-bottom" label="End Bottom"></vaadin-radio-button>
+    </vaadin-radio-group>
+
     <div id="wrapper">
       <div id="target">Save</div>
+      <vaadin-tooltip id="tooltip" text="Tooltip" position="bottom" for="target"></vaadin-tooltip>
```

Comment:
No need to set the `bottom` position explicitly as it's the default one:

```suggestion
      <vaadin-tooltip id="tooltip" text="Tooltip" for="target"></vaadin-tooltip>
```